### PR TITLE
test

### PR DESCRIPTION
**Pull request by alexandrumalinescu**

### CODE REVIEW

File: blogJs2/script.js [hunk 1]
```diff
@@ -223,6 +233,23 @@ function createArticle(article) {
...
 
+//-----delete article
+
+function deleteArticle(id) {
+  fetch("http://localhost:3001/article/" + id, {
+    method: "DELETE",
+  })
+    .then(json)
+    .then((data) => {
+      const a = "check test";
```
Comment (ghost) [inline blogJs2/script.js]: test comment

File: blogJs2/script.js [hunk 2]
```diff
@@ -223,6 +233,23 @@ function createArticle(article) {
...
+      console.log("delete succes", data);
+      window.location.hash = `#index.html`;
```
Comment (ghost) [inline blogJs2/script.js:245]: aaaaaaa

File: blogJs2/script.js (hunk 1 above)
Comment (alexandrumalinescu) [reply blogJs2/script.js]: i will delete that variable

File: blogJs2/script.js [hunk 3]
```diff
@@ -241,7 +241,6 @@ function deleteArticle(id) {
   })
     .then(json)
     .then((data) => {
-      const a = "check test";
```
Comment (ghost) [inline blogJs2/script.js]: nice work!

**Review by ghost — APPROVED** [review]

**Pull request merged**